src/chain.js

```diff
@@ -59,12 +59,19 @@ function startServer(options) {
       oldSendAsync(payload, callback)
     }
 
-    server.listen(options.port, options.hostname, function(err, state) {
+    server.listen(options.port, options.hostname, function(err, result) {
       if (err) {
         process.send({type: 'start-error', data: err});
         return
       }
 
+      var state = result ? result : server.provider.manager.state;
```

Comment:
This is a small workaround to an issue I accidentally introduced during the 6.x dev cycle (server callback isn't passed the state object anymore). With the way this is written fixing the problem won't break ganache.

I was dumb and released ganache-core before I found this issue (will test better next time), so rather than pushing out yet another release, I just created this workaround. Will fix the issue in the next ganache-core release.

Reply:
:+1: Thanks for the writeup!